web/src/components/atoms/core/index.tsx

```diff
@@ -1,3 +1,10 @@
+export { Button, ButtonProps } from './Button';
```

Comment:
Hmm, thinking here is that any files exporting typescript components should be `tsx`. Yes? No?

Reply:
both are typescript - it's just that files containing react elements have to be `.tsx` in order to have the elements parsed. I seem to recall that there's a minor overhead in doing this, and therefore any files that are strictly javascript should be `.ts` ( same deal with `.jsx` and `.js` )
I may be wrong - and it might just be a convention that I've fallen into. I find it helpful to easily distinguish between react element files and vanilla script files.

Reply:
I agree that plain typescript should be `ts`, but I think it's a good convention to use `tsx` for files which export components (as they technically contain react elements)?